### DISCUSSION TITLE
If I use this add-on then can I maximize the browser?

### DISCUSSION THREAD

**Discussion by asteria121** [kkapsner/CanvasBlocker#627] [Q&A]
Since this addon spoof that information so can I maximize the browser and no problem with it?

**Comment by kkapsner** [comment]
CanvasBlocker has no problem when you maximize the browser. But you then might be put in a different screen size bucket.

**Reply by asteria121** [reply]
"But you then might be put ina different size bucket."
Is that means when I maximize browser then my fingerprint can be identified as unique?

**Reply by kkapsner** [reply]
No - it still should use a fairly common size. Unless you have a extremely big screen (https://github.com/kkapsner/CanvasBlocker/blob/master/lib/modifiedScreenAPI.js#L42)
The details can also be found in the description of the "Fake minimal screen size" setting.

But you can also set the reported screen size manually (you have to disable "Fake minimal screen size" before and enable the expert mode).
